mozjs/build.rs

```diff
@@ -39,7 +39,7 @@ const EXTRA_FILES: &'static [&'static str] = &[
 ];
 
 /// Which version of moztools we expect
-const MOZ_TOOLS_VER: &str = "3.2";
+const MOZ_TOOLS_VERSION: &str = "3.2";
```

Comment:
Apologies for missing this, but:

```suggestion
const MOZTOOLS_VERSION: &str = "3.2";
```